### DISCUSSION TITLE
How to use PostCSS alongside TailwindCSS? Syntax errors in Visual Studio Code

### DISCUSSION THREAD

**Discussion by jamesvclements** [tailwindlabs/tailwindcss#11456] [Help] [answered]
I have the `tailwindcss/nesting` plugin installed alongside the standard Tailwind set-up.
It seems like the TailwindCSS plug-in for Visual Studio code doesn't like PostCSS syntax:

![globals css — paper-triangles-site-Monday-June-19-2023-11 51 09AM@2x](https://github.com/tailwindlabs/tailwindcss/assets/20052710/a234f07c-7ec9-44e7-932f-82e1c546f902)

Is there a way to tell the plug-in that I'm using `nesting` so it doesn't complain?

**Comment by wongjn** [comment, marked as answer]
Seems like you'd need to wait for VSCode's CSS language service supports it or try using PostCSS language service instead:

https://github.com/tailwindlabs/tailwindcss-intellisense/issues/764#issuecomment-1526883265
https://github.com/tailwindlabs/tailwindcss-intellisense/issues/539#issuecomment-1108709757

**Comment by jamesvclements** [comment]
@wongjn thank you! Spot-on. Do you know if there's a way to disable linting for just a specific line? Similar to prefixing a block with `@ts-ignore`.

**Reply by wongjn** [reply]
I don't know if there is a way.
